tests/src/core/parse.js

```diff
@@ -1,11 +1,14 @@
 import * as fs from 'fs'
 import { expect } from 'chai'
+import sinon from 'sinon'
 import parse from 'eslint-module-utils/parse'
 
 import { getFilename } from '../utils'
 
 describe('parse(content, { settings, ecmaFeatures })', function () {
   const path = getFilename('jsx.js')
+  const parseStubParser = require('./parseStubParser')
+  const parseStubParserPath = require.resolve('./parseStubParser')
```

Comment:
In other words, `require('./parseStubParser')` and `require.resolve('./parseStubParser')` should sit next to each other, they are closely related in this test.